projects/kit/src/lib/masks/time/utils/parse-time.ts

```diff
@@ -0,0 +1,34 @@
+import {DEFAULT_TIME_SEGMENT_MAX_VALUES} from '../../../constants';
+import type {MaskitoTimeSegments} from '../../../types';
+import {parseTimeString} from '../../../utils/time';
+import type {MaskitoTimeParams} from '../maskito-time-params';
+
+export function maskitoParseTime(
```

Comment:
Maybe it makes sense to add a parameter to ensure that the entire string is entered? Otherwise, `null` will be returned. @nsbarsukov

Reply:
I think one more parameter is not required.
Developer can easily check it by yourself:

```ts
const timeParams: MaskitoTimeParams = {mode: 'HH:MM:SS'};
const timeString = '23:59:9';
const isCompleteTimeString = timeString.length < timeParams.mode.length;
```